spec/models/transaction_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
 
 RSpec.describe Transaction, type: :model do
-  it {should belong_to(:invoice)}
+  it { should respond_to(:invoice) }
```

Comment:
Is respond to preferable to belong_to here? Respond to is more of an interface test than a relationship test: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/respond-to-matcher